elements/lisk-dpos/src/constants.ts

```diff
@@ -13,9 +13,10 @@
  */
 
 export const EVENT_ROUND_CHANGED = 'EVENT_ROUND_CHANGED';
-export const CONSENSUS_STATE_VOTE_WEIGHTS_KEY = 'dpos:voteWeights';
-export const CONSENSUS_STATE_FORGERS_LIST_KEY = 'dpos:forgersList';
-export const CHAIN_STATE_DELEGATE_USERNAMES = 'chain:usernames';
+// State store related constants
+export const CONSENSUS_STATE_VOTE_WEIGHTS = 'dpos:voteWeights';
+export const CONSENSUS_STATE_FORGERS_LIST = 'dpos:forgersList';
+export const CHAIN_STATE_DELEGATE_USERNAMES = 'chain:delegateUsernames';
```

Comment:
Either we prefix the same information to all or we can skip. e.g. vote weights also belong to delegates as similar to usernames. 

So either both `dpos:delegateVoteWeights`, `dpos:delegateUsernames`  or `dpos:voteWeights` and `dpos:usernames`

Reply:
ok, I am ok with this change `export const CHAIN_STATE_USERNAMES = 'chain:usernames'`.

Reply:
Yeah, that's fine either case. If you prefer more descriptive then can use other option with `delegateUsernames` and `delegateVoteWeights`